.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,13 @@
+---
+name: Blank issue (Bug, Feature request)
+about: Do you think found an actual bug? Or want to follow up a feature request from a community call? Go here.
```

Comment:
> Do you think found an actual bug?

A `you` is missing in this sentence.

> Or want to follow up a feature request from a community call?

The template will also be used for new feature requests, how about removing the parts about the community call?

What do you think about:
`You found a bug, or want to create a feature request? Go here.`

As this template is meant for bug reports and feature requests, the file should be renamed to represent this.

Reply:
@lenucksi, we prefer review comments to the addressed by the original author, by amending existing commits, not adding new ones.